### PR TITLE
Feature/redesign 2024

### PR DESCRIPTION
**Pull request by ubong-s**
Homepage design complete

### CODE REVIEW

File: src/pages/home-alt.astro
Comment (shippy) [inline src/pages/home-alt.astro:1]: Should probably remove this one?

File: src/pages/blog/index.astro [hunk 1]
```diff
@@ -1,4 +1,4 @@
----
+<!-- ---
```
Comment (shippy) [inline src/pages/blog/index.astro:1]: Is commenting out the whole page (including frontmatter) intentional?

File: src/pages/home-alt.astro
Comment (ubong-s) [reply src/pages/home-alt.astro:1]: Will do in the next commit

File: src/pages/blog/index.astro (hunk 1 above)
Comment (ubong-s) [reply src/pages/blog/index.astro]: Yes...but it's been fixed now

**Pull request merged**